test/templates/unit_test.tpl

```diff
@@ -17,6 +17,9 @@ jobs:
     - name: Tune the OS
       run: |
         echo '1024 65535' | sudo tee -a /proc/sys/net/ipv4/ip_local_port_range
+        # Increase the asynchronous non-blocking I/O. More information at https://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_use_native_aio
+        echo "\nfs.aio-max-nr = 1048576" >> /etc/sysctl.conf
```

Comment:
You need to use `echo -e` if you don’t simply want the \n characters printed but instead the new line escape character.

Reply:
fixed this by using the same way as in the line before it, using `tee`